content/docs/get-started/index.md

```diff
@@ -4,97 +4,214 @@ description: 'Learn how you can use MLEM to easily manage and deploy models'
+install all the requirements (including MLEM) there:
 
 ```cli
+$ mkdir mlem-get-started
+$ cd mlem-get-started
 $ python3 -m venv .venv
 $ source .venv/bin/activate
-$ pip install -r requirements.txt
+$ pip install pandas scikit-learn mlem[fastapi,heroku]
 ```
 
 </details>
 
-```cli
-$ mlem init
-```
+## Saving your ML model
 
-A few [internal files](/doc/user-guide/project-structure) will be created:
+To enable all kinds of productionization scenarios MLEM supports, we first need
+to save a ML model with MLEM.
 
-```cli
-$ tree .mlem
-.mlem
-└─── config.yaml
-```
+Let's take a look at the following Python script:
 
-Now you’re ready to MLEM!
+```py
+# train.py
+from sklearn.datasets import load_iris
+from sklearn.ensemble import RandomForestClassifier
 
-In our
-[example repository](https://github.com/iterative/example-mlem-get-started),
-you'll find tags for each step we take in the different sections of this
-tutorial. You can just see what is going on there or reproduce everything
-yourself and compare. In the different `Get Started` sections, those tags will
-be marked with ⛳ emoji. Click on it to expand the section and see the `git`
-commands to run if you are following along. Just like this Git tag that
-concludes this section:
+from mlem.api import save
 
-<details>
 
-# ⛳ MLEM init
+data, y = load_iris(return_X_y=True, as_frame=True)
+rf = RandomForestClassifier(
+    n_jobs=2,
+    random_state=42,
+)
+rf.fit(data, y)
+save(
+    rf,
+    "models/rf",
+    sample_data=data,
+)
+```
+
+Here we load well-known iris dataset with sklearn and train a simple classifier.
+But instead of pickling the model we saved it with MLEM (check out the
+[full list of ML frameworks](/doc/object-reference/model) that MLEM supports.)
 
-Tag:
-[1-mlem-init](https://github.com/iterative/example-mlem-get-started/tree/1-mlem-init)
+Now let's run this script and see how MLEM saved the model.
 
 ```cli
-$ git add .mlem
-$ git status
-Changes to be committed:
-        new file:   .mlem/config.yaml
-        ...
-$ git commit -m "Initialize MLEM"
+$ python train.py
+...
+
+$ tree models/
+models
+├── rf
+└── rf.mlem
 ```
 
-To compare your results with the tag you can also run the following
+The model was saved along with some metadata about it: `rf` is the model binary
+and `rf.mlem` is a metafile containing information about the model. Let's take a
+look at it:
 
-```cli
-$ git diff 1-mlem-init
-```
+<details>
 
-The output will be empty if you have the same files staged/committed
+### `$ cat models/rf.mlem`
+
+```yaml
+artifacts:
+  data:
+    hash: 5a38e5d68b9b9e69e9e894bcc9b8a601
+    size: 163651
+    uri: rf
+model_type:
+  methods:
+    predict:
+      args:
+        - name: data
+          type_:
+            columns:
+              - sepal length (cm)
+              - sepal width (cm)
+              - petal length (cm)
+              - petal width (cm)
+            dtypes:
+              - float64
+              - float64
+              - float64
```

Comment:
Do we need all the lines? Even if this is hidden, it's super long. Is it informative at all? Can we truncate some parts to highlight what matters most?

Reply:
Good point, made it shorter a bit.